src/locale/sv-SE.h

```diff
@@ -0,0 +1,189 @@
+// Copyright 2021 - Tom Rosenback (@tomrosenback)
+// Locale/language file for swedish / Sweden.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_SV_SE_H_
+#define LOCALE_SV_SE_H_
+
+#define D_STR_UNKNOWN "OKÄND"
+#define D_STR_PROTOCOL "Protokoll"
+#define D_STR_POWER "Strömläge"
+#define D_STR_PREVIOUS "Föregående"
+#define D_STR_ON "På"
+#define D_STR_OFF "Av"
+#define D_STR_MODE "Läge"
+#define D_STR_TOGGLE "Toggla"
```

Comment:
Maybe "Toggel" or "Växla" might be better here when considering how it is used in text.

Reply:
Växla is probably better then, changing.